release-notes/elastic-security/index.md

```diff
@@ -27,6 +27,105 @@ To check for security updates, go to [Security announcements for the Elastic sta
 
 % *
 
+
+## 9.2.0 [elastic-security-9.2.0-release-notes]
+
+### Features and enhancements [elastic-security-9.2.0-features-enhancements]
+
+* Adds the Security Entity Analytics risk score reset feature [#237829]({{kib-pull}}237829).
+* Introduces a Security risk scoring AI Assistant tool [#233647]({{kib-pull}}233647).
+* Uses {{esql}} for calculating entity risk scores [#237871]({{kib-pull}}237871).
+* Updates the entity source saved object schema to support integrations sync markers [#236457]({{kib-pull}}236457).
```

Comment:
This one is an implementation detail of `#237129` a few lines below, I think we should consider removing it